modules/entity_version_history/tests/FunctionalJavascript/HistoryTabSettingsTest.php

```diff
@@ -0,0 +1,117 @@
+    $history_bundle_checkbox->check();
+    $page->pressButton('Save configuration');
+
+    $status_message = $assert_session->waitForElement('css', '.messages--status');
+    $this->assertEquals($status_message->getText(), 'Status message Configuration has been saved.');
+
+    // Make sure that the settings are reflected in the form.
+    $this->assertTrue($node_entity_checkbox->isChecked());
+    $this->assertTrue($first_bundle_checkbox->isChecked());
+    $this->assertTrue($second_bundle_checkbox->isChecked());
+    $this->assertTrue($history_entity_checkbox->isChecked());
+    $this->assertTrue($history_bundle_checkbox->isChecked());
+
+    // Make sure configuration saved correctly and complies with the schema.
+    $config_data = $this->config('entity_version_history.settings.node.first_bundle');
+    $this->assertEquals($config_data->get('target_entity_type_id'), 'node');
+    $this->assertEquals($config_data->get('target_bundle'), 'first_bundle');
+    $this->assertEquals($config_data->get('target_field'), 'field_entity_version');
+    $this->assertConfigSchema(\Drupal::service('config.typed'), $config_data->getName(), $config_data->get());
+
+    $config_data = $this->config('entity_version_history.settings.node.second_bundle');
+    $this->assertEquals($config_data->get('target_entity_type_id'), 'node');
+    $this->assertEquals($config_data->get('target_bundle'), 'second_bundle');
+    $this->assertEquals($config_data->get('target_field'), 'field_entity_version');
+    $this->assertConfigSchema(\Drupal::service('config.typed'), $config_data->getName(), $config_data->get());
+
+    $config_data = $this->config('entity_version_history.settings.history_entity_test.history_entity_test');
+    $this->assertEquals($config_data->get('target_entity_type_id'), 'history_entity_test');
+    $this->assertEquals($config_data->get('target_bundle'), 'history_entity_test');
+    $this->assertEquals($config_data->get('target_field'), 'version');
+    $this->assertConfigSchema(\Drupal::service('config.typed'), $config_data->getName(), $config_data->get());
+
+    // Remove the config by unchecking the entity checkbox.
+    $history_bundle_checkbox->uncheck();
+    $page->pressButton('Save configuration');
+
+    // Make sure that the settings are reflected in the form.
+    $this->assertTrue($node_entity_checkbox->isChecked());
+    $this->assertTrue($first_bundle_checkbox->isChecked());
+    $this->assertTrue($second_bundle_checkbox->isChecked());
+    $this->assertFalse($history_entity_checkbox->isChecked());
+    $this->assertFalse($history_bundle_checkbox->isChecked());
+
+    // Check the config is deleted.
+    $this->container->get('config.factory')->clearStaticCache();
+    $config_data = $this->config('entity_version_history.settings.history_entity_test.history_entity_test');
+    $this->assertNull($config_data->get('target_entity_type_id'), 'history_entity_test');
```

Comment:
What is the second paremeter of this assertion? The message? `history_entity_test `?